activerecord/test/cases/tasks/sqlite_rake_test.rb

```diff
@@ -15,6 +15,9 @@ def setup
       File.stubs(:exist?).returns(false)
       ActiveRecord::Base.stubs(:connection).returns(@connection)
       ActiveRecord::Base.stubs(:establish_connection).returns(true)
+
+      @original_stdout = $stdout
```

Comment:
Set `$stdout = @original_stdout` in teardown to ensure we don't hijack stdout for other tests.